sensor_msgs/NavSatFix[] gps_wp
geometry_msgs/Quaternion[] orientation
---
std_msgs/Bool success
